docs/hugo.toml

```diff
@@ -0,0 +1 @@
+theme = ['doks']
```

Comment:
This can get added to the main config/_default/config.toml file and this file can be removed.

Reply:
Nice, removed!